hs_access_control/models.py

```diff
@@ -1277,6 +1277,23 @@ def can_change_resource_flags(self, this_resource):
 
         return self.user.is_superuser or (not this_resource.raccess.published and self.owns_resource(this_resource))
 
+    def can_create_resource_version(self, this_resource):
+        """
+        Whether self can create a version of a resource.
+
+        :param this_resource: Resource to check.
+        :return: True if user can set create a version otherwise false.
+
+        This is not enforced. It is up to the programmer to obey this restriction.
+        """
+
+        if __debug__:  # during testing only, check argument types and preconditions
+            assert isinstance(this_resource, BaseResource)
+
+        if not self.user.is_active: raise PermissionDenied("Requesting user is not active")
+
+        return self.owns_resource(this_resource)
+
```

Comment:
@pkdash Understood the reason why you add this function, but since we don't have any additional conditions at this point in time other than just allowing owners to version a resource, I think it is good just to remove this function, but use owns_resource() to check for permissions for resource versioning for now. If we need to add more conditions in the future, we can always add new function in, but it is very possible this will never be needed, which is why I prefer just to remove this function for now.